index.js

```diff
@@ -1,22 +1,26 @@
+'use strict';
+
 var request = require('request')
   , qs = require('querystring')
   , jsonreq = request.defaults({json:true})
-  ;
 
-var chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+function makeError(err, resp){
+  var errObject = new Error(resp.statusCode + ' ' + err.reason)
```

Comment:
the assumes that error is both not `null`, and an `Error`, which looking at the rest of the code, isn't necessarily true.